opal/util/net.c

```diff
@@ -405,7 +403,6 @@ opal_net_get_hostname(const struct sockaddr *addr)
        int err = errno;
        opal_output (0, "opal_sockaddr2str failed:%s (return code %i)\n",
                     gai_strerror(err), error);
-       free (name);
```

Comment:
To be similar to inet_ntoa I guess (the returned string is in a static area that is overwritten by successive calls).